examples/mouse_eye_movements.py

```diff
@@ -0,0 +1,300 @@
+    for coord, color in ["x", "C0"], ["y", "C1"]:
+        axes[i].plot(
+            time_points,
+            da.sel(space=coord),
+            label=coord,
+            color=color,
+        )
+    axes[i].set_xlabel("Time (s)")
+    axes[i].set_ylabel("Position")
+    axes[i].set_title(f"Pupil centroid position ({da_name})")
+    axes[i].legend()
+
+for ax in axes.flat:
+    ax.set_ylim(-30, 65)
+
+plt.tight_layout()
+plt.show()
+# %%
+# Pupil velocity over time
+# ------------------------
+# We use ``compute_velocity`` from the ``movement.kinematics`` module to
+# calculate the velocity with which the pupil centroid moves.
+velocity_dict = {}
+for da_name, da in position_norm_dict.items():
+    velocity_dict[da_name] = kin.compute_velocity(da.sel(keypoints="pupil-C"))
+
+# %%
+# Now we can plot pupil velocity over time.
+fig, axes = plt.subplots(2, 1, figsize=(6, 4))
+for i, (da_name, da) in enumerate(velocity_dict.items()):
+    for coord, color in ["x", "C0"], ["y", "C1"]:
+        axes[i].plot(
+            time_points,
+            da.sel(space=coord, time=time_points),
+            label=coord,
+            color=color,
+        )
+    axes[i].set_xlabel("Time (s)")
+    axes[i].set_ylabel("Velocity")
+    axes[i].set_title(f"Pupil velocity ({da_name})")
+    axes[i].legend()
+
+plt.tight_layout()
+plt.show()
+# %%
+# The positive peaks correspond to rapid eye movements to the right, the
+# negative peaks correspond to rapid eye movements to the left.
+
+# %%
+# Pupil diameter
+# --------------
+# Here we define the pupil diameter as the distance between the two pupil
+# keypoints. We use ``compute_pairwise_distances`` from ``movement.kinematics``
+# to calculate the Euclidean distance between "pupil-L" and "pupil-R".
+pupil_diameter_dict = {}
+for da_name, da in position_norm_dict.items():
+    pupil_diameter_dict[da_name] = kin.compute_pairwise_distances(
+        da, "keypoints", {"pupil-L": "pupil-R"}
+    )
+
+
+# %%
+# A function to plot the pupil diameter is created so that we can easily
+# compare the effect of different filters.
+def plot_pupil_diameter(da_dict, **kwargs):
+    fig, ax = plt.subplots(figsize=(5, 3))
+    for da_name, da in da_dict.items():
+        x = da.sel(**kwargs).time  # time points
+        y = da.sel(**kwargs)  # pupil diameter
+        ax.plot(x, y, label=da_name)
+        ax.set_xlabel("Time (s)")
+        ax.set_ylabel("Pupil diameter (pixels)")
+    ax.legend()
+    ax.set_title("Pupil diameter")
+    plt.tight_layout()
+    return fig, ax
+
+
+# %%
+# Now the pupil diameter before filtering can be plotted.
+
+fig, ax = plot_pupil_diameter(pupil_diameter_dict)
+fig.show()
+# %%
+# The plot of the pupil diameter looks noisy. The very steep peaks are
+# unlikely to represent real changes in the pupil size. Unlike eye movements,
+# which can be extremely fast, changes in pupil size do not occur this fast.
+# Filters can be applied to reduce noise and make underlying trends clearer.
```

Comment:
I also suspect that some of the bigger spikes correspond to the eye-lids closing.
If you can confirm that, it's worth mentioning here as an explanation.